src/System.Xml.XPath.XmlDocument/src/System.Xml.XPath.XmlDocument.builds

```diff
@@ -3,6 +3,9 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="System.Xml.XPath.XmlDocument.csproj" />
+    <Project Include="System.Xml.XPath.XmlDocument.csproj">
```

Comment:
Issue was introduced with recent XML refactoring.  This assembly contains a single type that isn't inbox in desktop.  In 1.0 this assembly built against public API which could be shared with desktop. Now it's building against System.Private.Xml which cannot be shared with desktop.